src/Tribe/Editor.php

```diff
@@ -410,11 +410,15 @@ public function assets() {
 		 *
```

Comment:
I suggest updating the `@since` to call out the changes you made.

Reply:
@redscar good idea, I'll update that.